src/kudu/consensus/leader_election.cc

```diff
@@ -392,7 +392,7 @@ FlexibleVoteCounter::IsMajoritySatisfiedInRegions(
 
   for (const std::string& region : regions) {
     if (region.empty()) {
-      results.push_back(std::move(std::make_pair<>(false, false)));
+      results.push_back(std::make_pair<>(false, false));
```

Comment:
`std::move` turns an expression into a r-value. If the expression is already an r-value it's not very useful. It however could confuse the compiler into not doing copy elision. This is a compiler warning error in fbcode.